1-starter-app/AzureTablesDemoApplication/src/main/java/com/azure/samples/tablesapp/untils/WeatherDataUtils.java

```diff
@@ -0,0 +1,147 @@
+package com.azure.samples.tablesapp.untils;
```

Comment:
This file is missing the copyright notice.

Reply:
Sorry, I missed fix this file. Now fixed in the new version.